Insights/Account/WindowsSigninActivity.yaml

```diff
@@ -209,64 +209,71 @@ Insights:
  AdditionalQuery:
   Text: "See all Windows sign-ins"
   Query: "where RelatedRowSet =~ 'AllEvents' | where EventID in (4624,4625,4672) | extend SubjectUserName = columnifexists('SubjectUserName', 'EventDoesNotContain') | summarize Total= count(), max(HourOfLogin), min(HourOfLogin), historical_DayofWeek=make_set(DayofWeek), StartTime=max(StartTime), EndTime = min(StartTime), SourceIP = make_set(IpAddress), SourceHost = make_set(WorkstationName), SubjectUserName = make_set(SubjectUserName), HostLoggedOn = make_set(Computer) by Activity, TargetDomainName, TargetUserName, ProcessName, LogonTypeName | extend NumberOfHostsLoggedOn = array_length(HostLoggedOn)"
+
 Activities:
      EnabledByDefault: true
-     Title: "{{LogonTypeName}} log-ins to a host"
-     Content: "The user {{v_Account_Name}} logged on to host {{WorkstationName}} {{Count}} time(s)"
      Items:
         - Id: 0d4ec12e-e44a-40a4-bb87-3db84d2a8057
-          Title: "{{LogonTypeName}} log-ins to a host"
-          Content: "The user {{v_Account_Name}} logged on to host {{WorkstationName}} {{Count}} time(s)"
+          Name: "Interactive log-ins to a host"
+          Title: "Interactive log-ins to a host"
+          Content: "The user {{Account_Name}} logged on to host {{Computer}} {{Count}} time(s)"
           Description: "This activity lists the user's interactive log-ins grouped by Host."
           QueryDefinitions:
-            Filter: "where RelatedRowSet == 'UserSigninToSystems' and LogonTypeName == '2 - Interactive' | extend TimeGenerated=StartTime"
+            Filter: "where RelatedRowSet == 'AllEvents' and EventID==4624 and LogonTypeName == '2 - Interactive' | extend TimeGenerated=StartTime"
```

Comment:
I cannot exactly test this, but I think you are opening yourself for many, many activities firing.  I went with the summarized data in the 'UserSigninToSystems' to avoid firing too many Activities in the Timeline.

Reply:
Unfortunately, I don't think I can avoid it. The others (UserSigninToSystems) are already summarized and lost their TimeGenerated attribute. In any case, I do not expect floods since the summarizeBy limits the number of results to the number of defined bins (6 by default)

Reply:
Okay, we can revisit